dcadec.c

```diff
@@ -103,7 +103,7 @@ static char *make_spkr_mask_str(int mask)
         return "???";
 
     buf[0] = 0;
-    for (int i = 0; i < 16; i++) {
+    for (size_t i = 0; i < sizeof(spkr_pair_names)/sizeof(char *); i++) {
```

Comment:
Thanks guys.
Since this is the only place where the array length is being determined so far, I'm not adding a macro for now.